qiskit_experiments/library/randomized_benchmarking/rb_analysis.py

```diff
@@ -70,15 +319,17 @@ def _default_options(cls):
         """Default analysis options.
 
         Analysis Options:
-            error_dict (Dict[Tuple[Iterable[int], str], float]): Optional.
-                Error estimates for gates from the backend properties.
-            epg_1_qubit (Dict[int, Dict[str, float]]) : Optional.
-                EPG data for the 1-qubit gate involved,
-                assumed to have been obtained from previous experiments.
-                This is used to estimate the 2-qubit EPG.
-            gate_error_ratio (Dict[str, float]): An estimate for the ratios
-                between errors on different gates.
-
+            gate_error_ratio (Optional[Dict[str, float]]): A dictionary with gate name keys
+                and error ratio values used when calculating EPG from the estimated EPC.
+                The default value will use standard gate error ratios.
+                If set to ``False`` EPG will not be calculated.
+            gate_counts_per_clifford (Union[bool, Dict[str, float]]): A dictionary
```

Comment:
Looking over the code more, it seems like this shouldn't be an analysis option since it is never expected that a user should set this to a specific value. When needed it is always calculated via the stored circuit metadata, so its just using an analysis option to store an internal variable state (which I think you had originally but I complained about in a previous review 😇 )

If thats the case probably we should just make it an attribute of the class (though I would greatly prefer if we didn't use state to store temporary variable per analysis run, but for now I don't think there is a way around this with the current CurveAnalysis class)

Reply:
Done in https://github.com/Qiskit/qiskit-experiments/pull/762/commits/357f0a44944a6dce80df3a3b09a451b935045238